### DISCUSSION TITLE
p-password toggle-mask LTR design

### DISCUSSION THREAD

**Discussion by AyaHaniRahal** [primefaces/community#1474] [PrimeNG] [answered]
In RTL design the eye icon is overlapped by text input

![image](https://github.com/primefaces/community/assets/129728771/d44b6491-def3-49c1-bf34-ae5b65007cb7)
![image](https://github.com/primefaces/community/assets/129728771/9a778027-4ec9-4d16-8be9-5488a88647af)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

PrimeNG components don't have RTL support yet. But it's in our roadmap for future development.

**Comment by AyaHaniRahal** [comment]
Hi,
    ok, Thanks for your answer.
________________________________
